apps/SKonsole/Commands/PRCommand.cs

```diff
@@ -100,8 +122,21 @@ private static async Task RunPullRequestDescription(CancellationToken token, ILo
         string output = process.StandardOutput.ReadToEnd();
         var pullRequestSkill = kernel.ImportSkill(new PRSkill.PullRequestSkill(kernel));
 
-        var kernelResponse = await kernel.RunAsync(output, token, pullRequestSkill["GeneratePR"]);
+        var contextVariables = new ContextVariables(output);
+        contextVariables.Set("outputJson", outputJson.ToString().ToLowerInvariant());
```

Comment:
I'd prefer to have this be a general formatting message|instruction that is given to the prompt, to allow flexibility and enabling scenarios like #8 as well.

Reply:
Good point, added that